### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by Sahil25061999**
added:
product-list
filter

### CODE REVIEW

File: product_listing.css [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  --slider-color: rgba(0, 0, 0, 0.418);
+  --wishlist-active: rgb(255, 38, 38);
+}
+
+.body {
+  margin-top: 6rem;
+  height: 100vh;
+  width: 100vw;
+  display: grid;
+  grid-template-columns: 250px 1fr;
```
Comment (manish-GH) [inline product_listing.css]: you could probably use rem here

**Comment by prankurpandeyy** [comment]
looks good to me , please reduce the height of footer rest is good

**Review by prankurpandeyy — APPROVED** [review]

**Review by manish-GH — COMMENTED** [review]
You could probably reduce the height of the navbar for small screens. Add something to the bottom of the navbar like a border or box-shadow or give the navbar a different color to differentiate it from the main content.
Definitely add either a box-shadow or border for your cards.

Everything else looks good

File: product_listing.css (hunk 1 above)
Comment (Sahil25061999) [reply product_listing.css]: i am kind of going for that kind of look, converted the unit to rem

**Review by sri-rishi — APPROVED** [review]
Except changes mentioned above looks good to merge

**Pull request merged**